pkg/geo/parse.go

```diff
@@ -239,6 +239,9 @@ func parseGeoHash(g string, precision int) (geohash.Box, error) {
 		return geohash.Box{}, pgerror.Newf(pgcode.InvalidParameterValue, "length of GeoHash must be greater than 0")
 	}
 
+	// In PostGis, upper case letters are parsed as lower case letters.
```

Comment:
nit: capitalise PostGIS as PostGIS
maybe use the term "case insensitive"

Reply:
Done.